ykman-gui/qml/PivGeneralErrorPopup.qml

```diff
@@ -7,11 +7,62 @@ InlinePopup {
 
     standardButtons: Dialog.Ok
 
+    function getDefaultMessage(resp) {
+        switch (resp.error) {
+        case 'bad_format':
```

Comment:
Now that this is centralised, the error names are too general. `bad_format` and  `blocked` can mean more than one thing.

Reply:
Yeah, I've been thinking that too. Shall I do another pass and update all of these error codes to be more specific?

Also, the current solution to that is the `messages` parameter which allows these to be individually overridden in each call.